### PR TITLE
Add support for installing nightlies in install script

### LINKED ISSUE #538: Support nightlies through installation script
Make it possible to install nightlies version using [Scarb installation script](https://docs.swmansion.com/scarb/download#install-via-installation-script)

### PR DESCRIPTION
**Pull request by szymmis**
Resolves #538

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -400,12 +399,27 @@ get_architecture() {
...
+    else
+      _ref="tag/v${_requested_version}"
+    fi
+  fi
 
   local _response
 
-  say "retrieving $_requested_version version from ${SCARB_REPO}..."
-  _response=$(ensure curl -Ls -H 'Accept: application/json' "${SCARB_REPO}/releases/${_requested_ref}")
+  say "${_repo}/releases/${_ref}"
```
Comment (maciektr) [inline install.sh:420]: Just making sure, is this intentional?

Comment (mkaput) [reply install.sh:420]: Yeah, I don't think so

**Review by mkaput — COMMENTED** [review]
Sanity check: have you tested all of this manually? :)

File: install.sh (hunk 1 above)
Comment (szymmis) [reply install.sh:420]: It's good to have a second pair of eyes 😅, totally unintentional, thanks for the catch!

File: install.sh [hunk 2]
```diff
@@ -488,4 +526,4 @@ add_local_bin_to_path() {
     "Run 'source ${_profile}' or start a new terminal session to use Scarb."
 }
 
-main "$@" || exit 1
+main "$@" || exit 1
```
Comment (mkaput) [inline install.sh:529]:
```suggestion
main "$@" || exit 1

```

**Review by mkaput — COMMENTED** [review]
Please also run `checkbashisms install.sh`. We technically do not target bash here :)

**Review by mkaput — APPROVED** [review]

**Pull request merged**